tests/framework/test_util.cpp

```diff
@@ -96,6 +96,18 @@ std::string get_env_var(std::string const& name, bool report_failure) {
 }
 #endif
 
+template <typename T>
+void print_list_of_t(std::string& out, const char* object_name, std::vector<T> const& vec) {
```

Comment:
Nit: This is where the previous naming convention I used is bad cause its inconsistent with JSON

This function really is a 'print_object_which_contains_a_vector' but the current name is fine. I'm making this comment to remind myself to fixup this stuff here...